README.md

```diff
@@ -1,8 +1,49 @@
-# Fractal integration for Drupal / Twig
-
-This module enables you to build a living style guide based on Fractal within
-Drupal.
-
+# Fractal integration for Twig projects
+
+This module enables you to build a living style guide based on Fractal components using the Twig templating language.
+It can either be used as a Drupal module or as a standalone Twig extension.
+
+## Installation
+Add the following in your project `composer.json` file:
+```json
+"repositories": [
+  {
+    "type": "vcs",
+    "url": "https://github.com/netzstrategen/drupal-twig_fractal"
```

Comment:
(also below)
```suggestion
    "url": "https://github.com/netzstrategen/twig_fractal"
```
I would prefer a hyphen instead of a underscore though.  Is there a technical reason speaking against this?

IIRC it was a underscore before, because a Drupal module would normally use underscores only… but that is trying to establish coherence where it is not necessary, so we can use a different name/separator if we want.

Reply:
Yes, I think the Drupal naming convention was the reason. I'm happy to go with a hyphen as well 👍